### ISSUE TITLE
[Performance , SEO and webPage Indexing Bug fix]

### ISSUE THREAD

**Issue by okerekechinweotito** [Open-Science-Community-Saudi-Arabia/OSCSA_Website#88]
### Description
   
❔ The description content of the meta tag on Index.html is empty. The 'alt' attribute of most img tags are also empty. These constitute a huge SEO and accessibility problem and would prevent Google Search Engine crawlers from properly indexing the site
   
### Expected behavior

✅ The Meta tag  description attribute should be properly filled  with relevant information
✅  The img tags should have a descriptive 'alt' attribute for proper accessibility

### Actual behavior
 
❌description property of the meta tag is empty
❌The img tags alt attribute is empty

### How to reproduce

1. Go to  https://web.dev/measure/
2. Type in https://osc-ksa.com/ in the space provided 
3. Click on Run Audit (This may take about 15 seconds)
4. Click on SEO 
5. Expand the items shown to view the report

### My personal set up

HP OMEN 15 ( 1TB SSD , 16GB RAM , WINDOWS 11)
VSCODE v1.72
Chrome Dev v108

**Comment by ladyami** [comment]
Hello Good Day are you working on the issue? 
I am also working on this same Issue let me know so I can work on something else

**Comment by okerekechinweotito** [comment]
@ladyami  This issue has been worked on.  Refer to the PR link. Thanks
